app/helpers/bgeigie_imports_helper.rb

```diff
@@ -2,20 +2,20 @@ module BgeigieImportsHelper
 
   def collect_bgeigie_logs(bgeigie_logs)
     bgeigie_logs.collect do |b|
-      point = {
+      {
         :lat => b.latitude, 
         :lng => b.longitude,
         :cpm => b.cpm
       }
     end
   end
 
-  def bgeigie_nav_li(status)
+  def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
```

Comment:
I would rather decrease to like 10. I think that the size is currently 15, which is quite complex.